recipe/meta.yaml

```diff
@@ -13,7 +13,7 @@ source:
 
 build:
   number: 0
-  script: python setup.py install --single-version-externally-managed --record record.txt
+  script: pip install --no-deps .
```

Comment:
#4 had two redundant fixes:

1. use raw setuptools entrypoints
2. override with conda entrypoints

I removed the setuptools part and left the conda part